Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp

```diff
@@ -190,53 +190,54 @@ void HacksWidget::SaveSettings()
 
 void HacksWidget::AddDescriptions()
 {
-  static const char TR_SKIP_EFB_CPU_ACCESS_DESCRIPTION[] =
-      QT_TR_NOOP("Ignore any requests from the CPU to read from or write to the EFB.\nImproves "
-                 "performance in some games, but might disable some gameplay-related features or "
-                 "graphical effects.\n\nIf unsure, leave this unchecked.");
+  static const char TR_SKIP_EFB_CPU_ACCESS_DESCRIPTION[] = QT_TR_NOOP(
+      "Ignores any requests from the CPU to read from or write to the EFB.\n\n"
+      "Improves performance in some games, but might disable some gameplay-related "
+      "features or graphical effects.\n\nIf unsure, leave unchecked.");
   static const char TR_IGNORE_FORMAT_CHANGE_DESCRIPTION[] = QT_TR_NOOP(
-      "Ignore any changes to the EFB format.\nImproves performance in many games without "
-      "any negative effect. Causes graphical defects in a small number of other "
-      "games.\n\nIf unsure, leave this checked.");
+      "Ignores any changes to the EFB format.\n\nImproves performance in many games "
+      "without any negative effect. Causes graphical defects in a small "
+      "number of other games.\n\nIf unsure, leave checked.");
   static const char TR_STORE_EFB_TO_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
-      "Stores EFB Copies exclusively on the GPU, bypassing system memory. Causes graphical defects "
-      "in a small number of games.\n\nEnabled = EFB Copies to Texture\nDisabled = EFB Copies to "
```

Comment:
This restructuring is unwise. The Enabled:/Disabled: subsections should be kept entirely separate from the description itself, as they are callbacks to older settings to assist users in the transition and the intention is to remove them at a later date some point after 6.0.